lib/ruby_lsp/requests/semantic_highlighting.rb

```diff
@@ -38,6 +40,11 @@ class SemanticHighlighting < BaseRequest
         default_library: 9,
       }.freeze, T::Hash[Symbol, Integer])
 
+      SPECIAL_RUBY_METHODS = T.let((Module.instance_methods(false) +
+        Kernel.methods(false) + Bundler::Dsl.instance_methods(false) +
```

Comment:
Since this feature now depends on `bundler`, we should also make it and/or Ruby 2.7 an explicit requirement as stated [here](https://github.com/Shopify/ruby-lsp/pull/162#issuecomment-1164243697).